AsyncDisplayKit/ASTableView.mm

```diff
@@ -47,8 +47,8 @@ @implementation _ASTableViewCell
 
 - (void)layoutSubviews
 {
-  [_delegate willLayoutSubviewsOfTableViewCell:self];
   [super layoutSubviews];
+  [_delegate willLayoutSubviewsOfTableViewCell:self];
```

Comment:
@appleguy Sweet! Will rename it. Makes more sense now that this was an intended change previously but I think this is the more coherent approach.